source/content/multizone-failover.md

```diff
@@ -47,7 +47,9 @@ Pantheon’s disaster response team will proactively notify you of a zone failur
 The object cache is not preserved after a site failover. This means you must ensure that the site can handle having its cache dropped under regular visitor traffic in order for the site to operate as expected in a failover scenario.
 
 To test in a Test Environment, click the **Clear Caches** button in the upper right hand corner of the Site Dashboard.
-You can also [connect to your Object Cache instance](/redis#use-the-redis-command-line-client) and run the `flushall` command to clear object cache via the Redis CLI.
+
+You can also [connect to your Object Cache instance](/object-cache#use-the-redis-command-line-client) and run the `flushall` command to clear object cache via the Redis CLI.
```

Comment:
is the command line client being renamed as well?

Reply:
Not today.